golint/golint.go

```diff
@@ -69,25 +73,61 @@ func exists(filename string) bool {
 }
 
 func lintFiles(filenames ...string) {
-	files := make(map[string][]byte)
+	files := make(map[string]struct{})
```

Comment:
I will comment the maps, but I do not think that map[string]bool is better. The value is never touched.